### PR TITLE
Fix auditor

### PR DESCRIPTION
**Pull request by listx**
These address #184 and #185.

### CODE REVIEW

**Comment by listx** [comment]
After this merges, I have to (1) promote the `cip-auditor` image to production (probably the `cip` image as well), and then (2) redeploy the auditor on Cloud Run.

File: lib/audit/auditor.go [hunk 1]
```diff
@@ -392,7 +392,7 @@ func (s *ServerContext) Audit(w http.ResponseWriter, r *http.Request) {
 	// If we can't find the source registry for this image, then reject the
 	// transaction.
 	if string(srcRegistry.Name) == "" {
-		msg := fmt.Sprintf("(%s) TRANSACTION REJECTED: could not determine source registry: %v", s.ID)
+		msg := fmt.Sprintf("(%s) TRANSACTION REJECTED: could not determine source registry: %v", s.ID, gcrPayload)
```
Comment (tpepper) [inline lib/audit/auditor.go]: Is there any chance of over-logging with gcrPayload?  Sharing something that shouldn't be in the log?

Comment (listx) [reply lib/audit/auditor.go]: No, the  GCR payload only has very sparse details --- the image path, digest, and action (e.g., INSERT, DELETE)

**Pull request merged**